server/client.go

```diff
@@ -50,6 +67,9 @@ func (c *Client) createNewApplication(url string, body interface{}, token string
 	if err != nil {
 		return nil, err
 	}
+	if c.logRequests {
+		c.logRequest("createNewApplication", req, true)
+	}
```

Comment:
`c.logRequest()` checks the value of `c.logRequests` so I don't think we need the if statement here

Reply:
Done :+1: